app/models/table/relator.rb

```diff
@@ -40,7 +40,8 @@ def preview_for(object)
       data = {
         id:         object.id,
         name:       object.name,
-        updated_at: object.updated_at
+        updated_at: object.updated_at,
+        auth_tokens: ::User.where(id: object.user_id).first.get_auth_tokens
```

Comment:
This is safer:

```ruby
auth_tokens: ::User.where(id: object.user_id).first.try(:get_auth_tokens)
```

This way, if the user is not found (for some other bug that might be hidden by the one you're solving) it won't break, but rather default to previous behaviour (no auth tokens)

Reply:
will do